pgdrive/envs/pgdrive_env.py

```diff
@@ -176,14 +181,17 @@ def lazy_init(self):
         # init vehicle
         v_config = self.config["vehicle_config"]
         self.vehicle = BaseVehicle(self.pg_world, v_config)
+        self.vehicles = {i: BaseVehicle(self.pg_world, v_config) for i in range(self.config["num_player"])}
 
         # for manual_control and main camera type
         if (self.config["use_render"] or self.config["use_image"]) and self.config["use_chase_camera"]:
             self.main_camera = ChaseCamera(
                 self.pg_world.cam, self.vehicle, self.config["camera_height"], 7, self.pg_world
             )
         # add sensors
-        self.add_modules_for_vehicle()
+        self.add_modules_for_vehicle(self.vehicle)
+        for i in range(self.config["num_player"]):
```

Comment:
The way to get the index of self.vehicles is dangerous, use `for k, v in self.vehicles.items()`